pkg/gofr/external_db_test.go

```diff
@@ -203,3 +204,26 @@ func TestApp_AddScyllaDB(t *testing.T) {
 		assert.Equal(t, mock, app.container.ScyllaDB)
 	})
 }
+
+func TestApp_AddArangoDB(t *testing.T) {
+	t.Run("Adding ArangoDB", func(t *testing.T) {
+		port := testutil.GetFreePort(t)
+		t.Setenv("METRICS_PORT", strconv.Itoa(port))
+
+		app := New()
+
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+
+		mock := container.NewMockArangoProvider(ctrl)
+
+		mock.EXPECT().UseLogger(app.Logger())
+		mock.EXPECT().UseMetrics(app.Metrics())
+		mock.EXPECT().UseTracer(otel.GetTracerProvider().Tracer("gofr-arango"))
+		mock.EXPECT().Connect()
+
+		app.AddArango(mock)
```

Comment:
```suggestion
		app.AddArangoDB(mock)
```

Based on the test namedl, you wanted to rename this method

Reply:
yups did rename this also